pyscriptjs/tests/integration/test_py_repl.py

```diff
@@ -1,7 +1,14 @@
 import platform
 
+import pytest
+
 from .support import PyScriptTest, skip_worker
 
+pytest.skip(
+    reason="FIX LATER: pyscript NEXT doesn't support the REPL yet",
```

Comment:
Yes. Doesn't mean we won't test in this repo though. If it's an official plugin we need to have tests for it, just not here but in another folder/somewhere else

Reply:
Sure, I didn't mean remove it, I meant tests for core should not include tests for plugins. If we bring these tests into the `pyscript.core` folder none of the plugins should be tested there as each plugin would be tested in its own folder/project.

Reply:
Just to clarify, the tests are not in `pyscript.core` yet. I've keept them in the old legacy `pyscript` folder on purpose to get them "working enough" first and have some visibility into behavior changes first and then decide where to put them later.

Reply:
reasonable ... but think "*once in Rome*" I'd love to have tests for current core in core so that any push to core tests core only things instead of waiting for all future things to run too (arguably desired anyway to be sure we don't break the rest) ... nothing that important thought, and actually having unified tests for any sub-project changes might be good!